config/routes.rb

```diff
@@ -1,6 +1,12 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  resources :merchants, only: :show do
+    scope module: :merchants do
+      resources :items, only: [:index, :show]
+    end
+  end
+
   resources :merchants, only: [:show] do
```

Comment:
Wouldn't this line duplicate the routes in line 3? Would we maybe want to embed line 10 inside the other structure?

Reply:
Updates look great!